profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -83,6 +86,8 @@ Map<String, String> defaultProperties() {
     config.put(CONFIG_KEY_MEMORY_ENABLED, String.valueOf(DEFAULT_MEMORY_ENABLED));
     config.put(CONFIG_KEY_MEMORY_EVENT_RATE, DEFAULT_MEMORY_EVENT_RATE);
     config.put(CONFIG_KEY_CALL_STACK_INTERVAL, DEFAULT_CALL_STACK_INTERVAL.toMillis() + "ms");
+
+    config.put(CONFIG_KEY_ENABLE_SNAPSHOT_PROFILER, "false");
```

Comment:
is this needed?

Reply:
I like that it's explicit and shows that the feature is not enabled by default. 🤷🏻